src/merge.cpp

```diff
@@ -49,15 +49,14 @@ Func merge_temporal(Image<uint16_t> imgs, Func alignment) {
 
     // constants for determining strength and robustness of temporal merge
 
-    float factor = 8.f;                         // factor by which inverse function is elongated
-    int min_dist = 10;                          // pixel L1 distance below which weight is maximal
-    int max_dist = 300;                         // pixel L1 distance above which weight is zero
+    int min_dist = 4;                          // pixel L1 distance below which weight is maximal
+    int max_dist = 128;                         // pixel L1 distance above which weight is zero

```

Comment:
What raw images are you using to adjust this tuning? I tuned for the Canon 5D III. I am hesitant to change tuning without testing on multiple raw, underexposed bursts, as intended. If you would like, I could send you raw images for testing.